### PR TITLE
#108: Fix Eclipse warnings in version 3.3.1

### LINKED ISSUE #108: Fix Eclipse warnings in version 3.3.1
## Situation

In version 3.3.1 we have 19 Eclipse warnings. Mostly about unclosed resources in integration tests.

## Acceptance Criteria

1. All warnings fixed.

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #108.

### CODE REVIEW

**Review by jakobbraun — CHANGES_REQUESTED** [review]
add to changelog

**Comment by redcatbear** [comment]
Added change log entry.

File: doc/changes/changes_3.4.0.md [hunk 1]
```diff
@@ -6,11 +6,12 @@ In the 3.4.0 release we added the ability to override the docker image used via
  
 ## Features
  
+* #105: Fixed code smells.
```
Comment (jakobbraun) [inline doc/changes/changes_3.4.0.md:9]:
```suggestion
* #108: Fixed code smells.
```

File: doc/changes/changes_3.4.0.md [hunk 2]
```diff
@@ -6,11 +6,12 @@ In the 3.4.0 release we added the ability to override the docker image used via
...
 * #106: Docker image can be overridden by property.
 
 ## Plugin Updates
 
 * Updated `org.jacoco:jacoco-maven-plugin:0.8.5` to `0.8.6`
 * Updated `org.codehaus.mojo:versions-maven-plugin:2.7` to `2.8.1`
 * Updated `com.exasol:exasol-jdbc:7.0.3` to `7.0.4`
-* Updated `org.mockito:mockito-junit-jupiter:3.6.0` to `3.6.28`
+* Updated `org.mockito:mockito-junit-jupiter:3.6.0` to `3.6.28`
```
Comment (jakobbraun) [inline doc/changes/changes_3.4.0.md:17]: use new format like for org.codehaus.mojo:versions-maven-plugin

**Review by jakobbraun — CHANGES_REQUESTED** [review]

**Review by jakobbraun — APPROVED** [review]

**Pull request merged**